### DISCUSSION TITLE
Missing basic multistore feature on PS 8.1

### DISCUSSION THREAD

**Discussion by cerodesign** [PrestaShop/PrestaShop#34233] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

 In new PS 8.1, you are not possible to use to add product to all stores, just to one of them. And than you need to add the produkt to another, and another, which completely degrate the multisore purpose (to add product just onetime and have it in all shops), since you have to add the product to every singe eshop!
![PS81](https://github.com/PrestaShop/PrestaShop/assets/42432086/ec473145-a27a-4c65-b0ae-faa0216f591b)

### Expected behavior

just to have ability to add product to all stores in one step. 

### Steps to reproduce

1. go to Catalog / Product
2. click to New product in editing All stores
3. and you get just possibility to add produkt or to 1 store, or to second store

### PrestaShop version(s) where the bug happened

8.1.1

**Comment by paulnoelcholot** [comment]
Hello @cerodesign 

I reproduce the issue with PrestaShop version 8.1.2, I'll add this to the backlog so it can be fixed.

Please be aware that some issues might take a very long time to be resolved. If this one is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you. 

If you fix the issue on your end, please [contribute it back to the project](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Thank you
[Products • the future (4).webm](https://github.com/PrestaShop/PrestaShop/assets/112863083/d1df22d7-0e8f-4854-b88f-a26166816a46)

**Comment by kpodemski** [comment]
Hello @cerodesign 

You can create a product in one store, and then you can click this icon and map your product to more stores:

https://capture.dropbox.com/WzOUvvMXwWd71NeO

@paulnoelcholot it is not a bug, it is a feature request.

I'm converting this issue into the Discussion.

CC @MatShir

**Comment by MatShir** [comment]
Close this conversation in favor of https://github.com/PrestaShop/PrestaShop/discussions/34225. Like this, the feedback are centralized one discussed.
